deseq_func.R

```diff
@@ -0,0 +1,42 @@
+
+# function to run deseq
+# se is a SummarizedExperiment object
+# col is the column of interest
+# ... can contain 0 or more adjustors the adjustors
+deseq_func <- function(se, col, ...) {
```

Comment:
this function name can be changed to, for example, `create_DESeqDataSet` because no deseq analysis is performed in this function yet

Reply:
Done